pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -56,9 +56,9 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/tools/record"
-	"k8s.io/cloud-provider"
+	cloudprovider "k8s.io/cloud-provider"
```

Comment:
Something I'm not understanding here. The file references cloudprovider. So if the import wasn't already aliasing cloud-provider to cloudprovider how was it compiling before your change? I would like to understand that to make sure we don't have a deeper issue.

Reply:
I learned this the other day, but apparently Go will ignore imports with `-`, so `k8s.io/cloud-provider` automatically becomes `cloudprovider`.

Reply:
oh no, it's actually because the package name is `cloudprovider` https://github.com/kubernetes/cloud-provider/blob/master/cloud.go#L17. And the package name does not have to match the repo name :)